### PR TITLE
Fix atmos devices

### PR DESCRIPTION
**Pull request by ThatDan123**
-Fixes newly placed atmos devices

-When you unwrench a device it'll now have the correct rotation too

-Just build testing now

### CODE REVIEW

File: UnityProject/Assets/Scripts/Objects/Directionals/Directional.cs [hunk 1]
```diff
@@ -122,6 +122,12 @@ public bool LockDirection
 		}
 	}
 
+	[ContextMenu("Current Direction")]
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Objects/Directionals/Directional.cs:125]: is this added to the right click menu?

**Review by corp-0 — DISMISSED** [review]

File: UnityProject/Assets/Scripts/Objects/Directionals/Directional.cs (hunk 1 above)
Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Objects/Directionals/Directional.cs:125]: nah, the directional component is terrible as you can't see what the current direction is even if you switch the outliner to debug mode. This just creates this:

image

File: UnityProject/Assets/Scripts/Objects/Pipes/PipeItemObject.cs [hunk 2]
```diff
@@ -24,7 +24,18 @@ public override void BuildPipe()
 
 			if (spawn.GameObject.TryGetComponent<Directional>(out var directional))
 			{
-				directional.FaceDirection(Orientation.GetOrientation(transform.localEulerAngles.z));
+				var orientation = Orientation.GetOrientation(transform.localEulerAngles.z);
+
+				if (orientation == Orientation.Up)
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Objects/Pipes/PipeItemObject.cs:30]: nani

Comment (corp-0) [reply UnityProject/Assets/Scripts/Objects/Pipes/PipeItemObject.cs:30]:
```suggestion
//TODO: find the cause for up and down being swaped and remove this hacky fix!
				if (orientation == Orientation.Up)
```

**Review by corp-0 — APPROVED** [review]

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**